Projects/Foundations/Schemas/Docs-Tutorials/I/Intro/Intro-to-Trading-Systems/intro-to-trading-systems-016-tutorial-step-ready-for-the-next-one.json

```diff
@@ -15,9 +15,9 @@
                 "updated": 1639148394256
             },
             {
-                "language": "TR",
-                "text": "Tebrikler! Artık ticaret ( Trading ) sistemleri hakkında benim kadar çok şey biliyorsun!",
-                "updated": 1640783928972
+                "language": "RU",
+                "text": "Поздравляю! Теперь вы знаете о торговых системах столько же, сколько и я! Далее - \"Определение торговой идеи\".",
+                "updated": 1640886497329
```

Comment:
All files from the first one having this issue until this one, about 16 Files are affected.